ray-operator/controllers/ray/common/pod_test.go

```diff
@@ -847,79 +847,68 @@ func TestBuildPod_WithCreatedByRayService(t *testing.T) {
 	utils.EnvVarExists(utils.RAY_TIMEOUT_MS_TASK_WAIT_FOR_DEATH_INFO, pod.Spec.Containers[utils.RayContainerIndex].Env)
 }
 
-func TestBuildPod_WithFeatureFlagLoginBash(t *testing.T) {
-	ctx := context.Background()
-	os.Setenv(utils.ENABLE_LOGIN_SHELL, "true")
-	defer os.Unsetenv(utils.ENABLE_LOGIN_SHELL)
-
-	cluster := instance.DeepCopy()
-	cluster.Spec.EnableInTreeAutoscaling = &trueFlag
-	podName := strings.ToLower(cluster.Name + utils.DashSymbol + string(rayv1.HeadNode) + utils.DashSymbol + utils.FormatInt32(0))
-	podTemplateSpec := DefaultHeadPodTemplate(ctx, *cluster, cluster.Spec.HeadGroupSpec, podName, "6379")
-	headPod := BuildPod(ctx, podTemplateSpec, rayv1.HeadNode, cluster.Spec.HeadGroupSpec.RayStartParams, "6379", true, utils.RayServiceCRD, "")
-
-	// make sure head container with login bash.
-	headContainer := headPod.Spec.Containers[utils.RayContainerIndex]
-	assert.Equal(t, []string{"/bin/bash", "-lc", "--"}, headContainer.Command)
-
-	// make sure autoscaler container with login bash.
-	index := getAutoscalerContainerIndex(headPod)
-	autoscalerContainer := headPod.Spec.Containers[index]
-	assert.Equal(t, []string{"/bin/bash", "-lc", "--"}, autoscalerContainer.Command)
-
-	worker := cluster.Spec.WorkerGroupSpecs[0]
-	podName = cluster.Name + utils.DashSymbol + string(rayv1.WorkerNode) + utils.DashSymbol + worker.GroupName + utils.DashSymbol + utils.FormatInt32(0)
-	fqdnRayIP := utils.GenerateFQDNServiceName(ctx, *cluster, cluster.Namespace)
-	podTemplateSpec = DefaultWorkerPodTemplate(ctx, *cluster, worker, podName, fqdnRayIP, "6379")
-	workerPod := BuildPod(ctx, podTemplateSpec, rayv1.WorkerNode, worker.RayStartParams, "6379", false, utils.RayServiceCRD, fqdnRayIP)
-
-	// make sure worker container with login bash.
-	workerContainer := workerPod.Spec.Containers[utils.RayContainerIndex]
-	assert.Equal(t, []string{"/bin/bash", "-lc", "--"}, workerContainer.Command)
-
-	// make init container with login bash.
-	initContainers := workerPod.Spec.InitContainers
-	for _, initContainer := range initContainers {
-		if initContainer.Name == "wait-gcs-ready" {
-			assert.Equal(t, []string{"/bin/bash", "-lc", "--"}, initContainer.Command)
-		}
+func TestBuildPod_WithLoginBash(t *testing.T) {
```

Comment:
There is no test to verify container command in `pod_test.go`, so I didn't delete this test.